app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def navbar_link(text, url)
+    button_classes = %w[button btn btn-default navbar-btn navbar-left]
+    button_classes << 'current' if current_page? url
```

Comment:
Not quite as fun as my solution! `current_page?(url) ? 'current' : ''` But I suppose it is easier to understand.